test/box-tap/gh-5602-environment-vars-cfg.test.lua

```diff
@@ -29,30 +29,42 @@ local repl1_sock = fio.pathjoin(fio.cwd(), 'repl1.sock')
 local repl2_sock = fio.pathjoin(fio.cwd(), 'repl2.sock')
 
 local cases = {
+    -- testcase 1
     ('%s %s %s %s %s'):format(
         string.format('TT_LISTEN=%s', listen_sock),
         'TT_READAHEAD=10000',
         'TT_STRIP_CORE=false',
         'TT_LOG_FORMAT=json',
         'TT_LOG_NONBLOCK=false'),
+
+    -- testcase 2
     ('%s %s %s %s'):format(
         string.format('TT_LISTEN=%s', listen_sock),
         string.format('TT_REPLICATION=%s,%s', repl1_sock,repl2_sock),
         'TT_REPLICATION_CONNECT_TIMEOUT=0.01',
         'TT_REPLICATION_SYNCHRO_QUORUM=\'4 + 1\''),
+
+    -- testcase 3
     'TT_BACKGROUND=true TT_VINYL_TIMEOUT=60.1',
+
+    -- testcase 4
     'TT_SQL_CACHE_SIZE=a',
+
+    -- testcase 5
     'TT_STRIP_CORE=a',
+
+    -- testcase 6
+    ('%s %s %s %s'):format(
+        string.format('TT_LISTEN=%s', listen_sock),
+        'TT_LOG=tarantool.log',
+        'TT_PID_FILE=tarantool.pid',
+        'TT_BACKGROUND=true'),
```

Comment:
Are these tests working? I mean I ran these tests on the master and they passed with no problem. If we remove these changes from the patch, the test will pass without problems. Should there be any effect?